modules/is_master/manifests/startserver.pp

```diff
@@ -0,0 +1,4 @@
+class is_master::startserver inherits is_master::params {
+
```

Comment:
Shall we add a comment here as well?

Reply:
What can we mention here?

Reply:
Explain the functionality of the class. Also, let's add the copyright notice as well.

Reply:
Resolved https://github.com/wso2/puppet-is/pull/58/commits/b8ba5cf3de87228146971ff913f98bd16021353b